src/main/java/uk/gov/hmcts/probate/changerule/ExecutorsRule.java

```diff
@@ -13,8 +13,18 @@
     @Override
     public boolean isChangeNeeded(CaseData caseData) {
         long numApplying = 0;
+
+
```

Comment:
Possibly, and this is just a maybe if you wanted to implement, we just check the lists that are applicable for title and clearing type selected.
This is to cover the scenario of a user checking a title / clearing option, and adding some executors, then going back and choosing a different title / clearing type. 
Or maybe its better as is, would force them to go back reselect the dodgy option, clear the executors then reselect the correct option!